File: src/main/java/com/mk/model/LegalClient.java
package com.mk.model;

import javax.persistence.Entity;
import javax.persistence.Table;

@Entity
@Table(name = "legal_client")
public class LegalClient extends Client {

	private String details;
	private String name;
	
	public LegalClient() {
	}

	public LegalClient(String fio, String phone, String adress, String details, String name) {
		super(fio, phone, adress);
		this.setDetails(details);
		this.setName(name);
	}

	public String getDetails() {
		return details;
	}

	public void setDetails(String details) {
		this.details = details;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
}

File: src/main/webapp/resources/js/app/model/LegalClient.js
Ext.define('app.model.LegalClient', {
    extend   : 'Ext.data.Model',
    fields : [
          {type: 'int',    name: 'id'},
          {type: 'string', name: 'fio'},
          {type: 'string', name: 'details'},
          {type: 'string', name: 'name'},
          {type: 'string', name: 'adress'},
          {type: 'string', name: 'phone'}

    ],
    idProperty: 'id',
    proxy: {
        type: 'ajax',
        api:{
            read:'lclient/list',
            update:'lclient/edit',
        }, 
        headers: {Accept:'application/json'},
        batchActions: false,
        reader: {type:'json',root:'entities'},
        writer: {type:'plainwriter'}
    }
});